core/src/templates/components/main-nav/main-nav.twig

```diff
@@ -18,39 +18,39 @@
   <button class="su-main-nav__toggle {{ toggle_modifier_class }}" aria-expanded="false">{{ toggle_text|default('Menu') }}</button>
   <ul class="su-main-nav__menu-lv1">
     {% spaceless %}
```

Comment:
I would rather not use `{% spaceless %}` around all the list items. It makes the generated markup very hard to read.

Reply:
I removed the spaceless tags and added a little bit of - sign trimming back in. Let me know if this is more readable.

Reply:
Yes, it is more readable. Thanks!